corehq/apps/app_manager/suite_xml.py

```diff
@@ -1407,11 +1416,26 @@ def case_sharing_requires_assertion(form):
                         return True
             return False
 
+        def updates_usercase(form):
+            actions = form.active_actions()
+            if 'update_case' in actions and hasattr(actions['update_case'], 'update'):
+                return any_usercase_items(actions['update_case'].update.keys())
+            return False
+
+        def preloads_usercase(form):
```

Comment:
Seems like you could combine these two functions since they only ever get used together